scapy/main.py

```diff
@@ -187,10 +187,9 @@ def init_session(session_name, mydict=None):
     global session
     global globkeys
     
-    scapy_builtins = importlib.import_module(".all", "scapy").__dict__
+    scapy_builtins = scapy_builtins = {k: v for k, v in six.iteritems(importlib.import_module(".all", "scapy").__dict__) if _validate_local(k)}
     for name, sym in six.iteritems(scapy_builtins):
-        if _validate_local(name):
-            six.moves.builtins.__dict__[name] = sym
+        six.moves.builtins.__dict__[name] = sym
```

Comment:
Good call moving the _validate_local outside of the loop. But now, we could replace this by a simple:
`six.moves.builtins.__dict__.update(scapy_builtins)`